geometry_msgs/Pose pickPos
---
bool isSetFinsh